### PR TITLE
Fixed MenAtPlay scene scraper

### PR DESCRIPTION
**Pull request by MortonBridges**
Previous xpath scene scraper for this URL was no longer working, likely due to changes in site design.  This version scrapes against the design used by the site in January 2022.

### CODE REVIEW

File: scrapers/MenAtPlay.yml [hunk 1]
```diff
@@ -4,24 +4,34 @@ sceneByURL:
...
+                        
       Studio:
-        Name:
-          fixed: MenAtPlay
-# Last Updated August 08, 2021
+         Name:
+           fixed: MenAtPlay
+           
+# Last Updated January 15, 2022
+
```
Comment (Belleyy) [inline scrapers/MenAtPlay.yml:37]: The last line should be the `Last Updated` one so remove the blank line.

File: scrapers/MenAtPlay.yml [hunk 2]
```diff
@@ -4,24 +4,34 @@ sceneByURL:
...
-        selector: //div[@class="video-descr__content"]
+         selector: //p[@class="update_description"]/text()
+
+      Performers:
+         Name:
+           selector: //div[@class="gallery_info spacer"]/p/span[@class="tour_update_models"]/a/text()
+        
+      Date:
+         selector: //span[@class="availdate"][1]
+         parseDate: Jan 2, 2006
```
Comment (bnkai) [inline scrapers/MenAtPlay.yml:23]: Haven't checked the selectors yet but you are using the old (deprecated) method for parseDate
It needs to be used inside a postProcess to pass validation
```yaml
         Date:
            selector: //span[@class="availdate"][1]
            postProcess:
               - parseDate: Jan 2, 2006
```

**Comment by bnkai** [comment]
@MortonBridges from what i understand you want 
<https://github.com/stashapp/CommunityScrapers/pull/859/commits/4221754219f68b79b1aa74718592e72c277111e6>
this ?
Move all the sites to the Mindgeek  scraper?

**Comment by MortonBridges** [comment]
> @MortonBridges from what i understand you want [4221754](https://github.com/stashapp/CommunityScrapers/commit/4221754219f68b79b1aa74718592e72c277111e6) this ? Move all the sites to the Mindgeek scraper?

That file was inadvertently added to the wrong directory.  I've deleted that errant file in my repo, and will attempt the changes again at a later time.

**Comment by bnkai** [comment]
@MortonBridges can you close and reopen the PR after merging or make a new pr with the latest updated part of the scaper?
I had to force-push the master branch and it has some conflicting files

**Comment by MortonBridges** [comment]
Closing per request.  Will resubmit at a later date.

**Pull request closed without merging**